API.md

```diff
@@ -33,6 +33,8 @@ in the [API.md](https://github.com/projectatomic/libpod/blob/master/API.md) file
 
 [func ImportImage(source: string, reference: string, message: string, changes: []string) string](#ImportImage)
 
+[func Info() PodmanInfo](#Info)
```

Comment:
I think the name should be GetInfo() more like GetVersion().

Reply:
ok